.github/workflows/github-ci.yml

```diff
@@ -292,16 +292,28 @@ jobs:
         #srun -M carrington -t 01:30:00 --job-name CI_testpackage --interactive --nodes=1 -c 4 -n 16 --mem-per-cpu=5G -p short ./small_test_carrington_github_ci.sh
         VLASIATOR_CIRUNNER_JOBSPEC_TP=$(sbatch --parsable -o testpackage_run_output.txt ./small_test_carrington_github_ci.sh)
         VLASIATOR_CIRUNNER_JOBSPEC_TP=(${VLASIATOR_CIRUNNER_JOBSPEC_TP//\;/ })
-        trap "scancel -M ${VLASIATOR_CIRUNNER_JOBSPEC_TP[1]} ${VLASIATOR_CIRUNNER_JOBSPEC_TP[0]}" SIGINT
-        srun -M carrington --dependency afterany:${VLASIATOR_CIRUNNER_JOBSPEC_TP[0]} echo "Job finished, checking output."
-        cat testpackage_run_output.txt
-        cat $GITHUB_STEP_SUMMARY > $GITHUB_WORKSPACE/testpackage_check_description.txt
-        cd $GITHUB_WORKSPACE
-        tar -czvf testpackage-output.tar.gz testpackage_check_description.txt testpackage_output_variables.txt
+        trap "scancel -M carrington ${VLASIATOR_CIRUNNER_JOBSPEC_TP[0]}" SIGINT
+        trap "scancel -M carrington ${VLASIATOR_CIRUNNER_JOBSPEC_TP[0]}" SIGTERM
+        RUN_STRING=$( cat << MORO
+        echo "Job finished, checking output." ;
+        cat testpackage_run_output.txt ;
+        cat $GITHUB_STEP_SUMMARY > $GITHUB_WORKSPACE/testpackage_check_description.txt ;
+        cd $GITHUB_WORKSPACE ;
+        tar -czf testpackage-output.tar.gz testpackage_check_description.txt testpackage_output_variables.txt ;
+        MORO
+        )
+        srun -M carrington --dependency afterany:${VLASIATOR_CIRUNNER_JOBSPEC_TP[0]} bash -c "$RUN_STRING"
         if [ -f $GITHUB_WORKSPACE/testpackage_failed ]; then
           # Fail this step if any test failed.
           exit 1
         fi
+    - name: Make sure the output binary is visible in lustre
```

Comment:
Suggestion: make the name correct. :) This is a tarball of the log file, not a binary file per se.
Perhaps the job could also print out the size of the log file, so we now when it's approaching 64kb?

Reply:
that's copy-pasted from above! :P